agent/app/agent_capability.go

```diff
@@ -136,6 +137,9 @@ func (agent *ecsAgent) capabilities() []*ecs.Attribute {
 			})
 		}
 	}
+	if agent.cfg.OverrideAWSLogsExecutionRole {
```

Comment:
Can you make a TODO and an issue to come back to this later?  We'll want to have non-overridden enabled with an appropriate Docker version once that ships.

Reply:
sure sounds good.